tools/mfassignr/mfassignr_recallist.xml

```diff
@@ -0,0 +1,47 @@
+<tool id="mfassignr_recallist" name="MFAssignR RecalList" version="@TOOL_VERSION@+galaxy0" profile="23.0">
+    <description>Identification of candidate series for recalibration.</description>
+    <macros>
+        <import>macros.xml</import>
+        <import>help.xml</import>
+    </macros>
+    <edam_topics>
+        <edam_topic>topic_3172</edam_topic>    
+    </edam_topics>
+    <edam_operations>  
+        <edam_operation>operation_3627</edam_operation>  
+    </edam_operations>
+    <expand macro="creator" />
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript '${run_script}'
+    ]]></command>
+    <configfiles>
+        <configfile name="run_script"><![CDATA[
+            df <- read.delim("$input_file", sep="\t")
+            recal <- MFAssignR::RecalList(
+                df = df
+            )
+
+            write.table(recal, file = '$recal_series', row.names= FALSE, sep = "\t")
+        ]]></configfile>
+    </configfiles>
+    <inputs>
+        <param name="input_file" type="data" format="tabular" label="Input data"
+               help= "Input data frame - output from MFAssign or MFAssignCHO. Recommended are 'Unambiguous' assignments."/>
+    </inputs>
+    <outputs>
+        <data name="recal_series" format="tabular" label="Recalibration series by ${tool.name} on ${on_string}"/>
```

Comment:
think about adding column names to your tabular files - will improve UX

Reply:
The tools are quite specific and most inputs are outputs from other tools, so I don't expect people to come with their own files. Or do you mean something else than adding the colnames to the help text?

Reply:
Something like that: https://github.com/galaxyproject/tools-iuc/blob/048d2859041c0bcc4be4e904e637a96fe564bd36/tools/onto_toolkit/get_term_id_vs_term_name.xml#L17

Reply:
This creates some very tight coupling between the package and the Galaxy tools :/